doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+========
+Tentacle
+========
+
+Tentacle is the 20th stable release of Ceph.
+
+v20.2.0 Tentacle
+================
+
+
+Highlights
+----------
+
+*See the sections below for more details on these items.*
+
+RADOS
+
+* FastEC: Long expected performance optimizations were added for EC pools.
```

Comment:
Suggest

* Long-anticipated performance and space amplification optimizations are added for erasure-coded pools

Reply:
@bill-scales wdyt about this phrasing?